docs/en/observability/whats-new.asciidoc

```diff
@@ -3,24 +3,101 @@
+Elastic Observability 8.14 announces the general availability (GA) of key Service Level Objective (SLO) management capabilities, additional enhancements to the Elastic AI Assistant for Observability, alerting improvements, and Universal Profiling for .NET.
 
+[float]
+== New SLO management capabilities
+
+[float]
+=== Federated SLOs over cross-cluster search (CCS)
+
+Administrators can now set up SLOs in local clusters and view them in a single "read" cluster using cross-cluster search. This improves performance in the read cluster, saves money by allowing transforms to be done closer to where the data lives, and provides an efficient way for our multi-cluster customers to centralize their SLO views.
+
+[role="screenshot"]
+image::images/wn-federated-1.png[What's new in 8.14: Screenshot of SLO overview page]
+
+[role="screenshot"]
+image::images/wn-federated-2.png[What's new in 8.14: Screenshot of Timeslice metric remote]
+
+[float]
+===  Grouped SLOs embeddable
+The SLO Overview chart can now show a group of SLOs. Multiple SLOs can be shown in a dashboard, grouped by SLO status, tag, or SLI type.
+
+[role="screenshot"]
+image::images/wn-grouped-1.png[What's new in 8.14: Screenshot of SLO configuration]
+
+[role="screenshot"]
+image::images/wn-grouped-2.png[What's new in 8.14: Screenshot of a visualization that identifies healthy and violated SLOs]
+
+[float]
+=== SLO burn rate alert dependency notification
+
+When an SLO is starting to burn through its error budget, but it is because of another  service that it is dependent on, you want to make sure that the notifications are sent for the dependent service but not for everyone else. SLO burn rate alerts can be noisy for end-to-end services that have service dependencies that are at the root cause. Now in 8.14, you can set up the SLO burn rate alerts to suspend the notification of burn rate alerts if there is a dependent service that is the root cause.
+
+[role="screenshot"]
+image::images/wn-slo-burn-rate.png[What's new in 8.14: Screenshot of the edit rule prompt]
+
+[float]
+=== New Synthetics availability SLI
+
+Introducing a new SLI dedicated to Synthetics availability. This will make it easier to convert your Elastic Synthetic monitors to availability SLOs with the same options that were used to set up the monitor. Synthetics is a great data source for availability monitoring.
+
+[role="screenshot"]
+image::images/wn-new-synthetics.png[What's new in 8.14: Screenshot of the Edit SLO page]
+
+[float]
+== Elastic AI Assistant for Observability enhancements
+
+[float]
+=== AI Assistant context-based starter prompts
+
+We added starter prompts to help new users with their onboarding on the AI Assistant. Default prompts can be overridden by contextual ones set by the current application. Starter prompts provide hints to the user on how to use the AI Assistant like "Are there any open alerts?" These prompts can be application- or data-specific.
```

Comment:
This paragraph has a few problems:
- We are really overloading the term "onboarding" at Elastic these days. Is this simply saying that the starter prompts help new users learn to use the AI Assistant?
- It's not clear who is doing the action here: "Default prompts can be overridden by contextual ones set by the current application." Is this simply saying that you can override the default prompts? 
- This sentence should probably come after the first sentence where we introduce the concept of starter prompts: "Starter prompts provide hints to the user on how to use the AI Assistant like "Are there any open alerts?" 
- I'm not exactly sure what this sentence means: "These prompts can be application- or data-specific."  I'd elaborate, provide more examples, or just remove this sentence.